lib/build.js

```diff
@@ -16,6 +16,7 @@ const tokenGen = Symbol('tokenGen');
 const uiUri = Symbol('uiUri');
 const ABORT_CODE = 130; // 128 + SIGINT 2 (^C)
 const TEMPORAL_JWT_TIMEOUT = 12 * 60; // 12 hours in minutes
+const STAGE_SETUP_PATTERN = /^stage@([\w-]+)(?::setup)$/; // stage@jobName:setup;
```

Comment:
This is a duplicate of https://github.com/screwdriver-cd/models/blob/master/lib/buildFactory.js#L10

Can we move the regex and the logic to determine if the job is stage `setup` to `helper.js`?

Reply:
updated